pup-0001.md

```diff
@@ -85,31 +87,29 @@ additional commits to the pull request either as patches or additional pull requ
 
 ### Voting
 
+The lazy consensus model is used for deciding via votes on the mailing list.
+
 The +1, +0, -0, -1 voting process is used; votes are sent as responses via the
-announcement/discussion mailing list. The voting has the following meanings:
+announcement/discussion thread on the mailing list. The voting has the following meanings:
 
 * +1: "Will benefit the project and should definitely be adopted."
 * +0: "Might benefit the project and is acceptable."
 * -0: "Might not be the right choice but is acceptable."
-* -1: "Not the right choice and should definitely not be adopted." This type of feedback must come
-  with a statement of objective concerns explaining what harm will come from adopting the proposal.
-
-Note that votes can come from anyone, not just core devs [1]. This is directly modeled after [the
-Django process](https://docs.djangoproject.com/en/dev/internals/contributing/bugs-and-features/#how-we-make-decisions)
-who modeled their process after the Python and Apache communities.
+* -1: "Not the right choice and should definitely not be adopted."
```

Comment:
What's the reasoning behind this change? I kind of like the requirement that a -1 vote must include an explanation and that we specify that anyone can vote.